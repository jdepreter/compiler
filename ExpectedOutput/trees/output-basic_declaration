digraph {
	1 [label=C]
	3 [label=definition]
	1 -> 3
	4 [label=int]
	3 -> 4
	5 [label=c]
	3 -> 5
	7 [label=assignment]
	1 -> 7
	8 [label=c]
	7 -> 8
	16 [label=2]
	7 -> 16
}
