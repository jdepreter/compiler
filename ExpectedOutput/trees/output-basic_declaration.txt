digraph {
	1 [label=C]
	3 [label=definition]
	1 -> 3
	4 [label=int]
	3 -> 4
	5 [label=c]
	3 -> 5
}
